pkg/query/api/v1.go

```diff
@@ -141,6 +141,7 @@ type API struct {
 	enableQueryPartialResponse bool
 	enableRulePartialResponse  bool
 	replicaLabels              []string
+	storeAllowList             []string
```

Comment:
Let's be explicit here. What about?
```suggestion
	storeAddressAllowlist             []string
```